operatorcert/entrypoints/reserve_operator_name.py

```diff
@@ -34,13 +34,11 @@ def setup_argparser() -> argparse.ArgumentParser:
 
 
 def check_operator_name(args) -> None:
-    rsp = get_with_cert(
+    rsp = pyxis.get(
         urljoin(
             args.pyxis_url,
             f"v1/operators/packages?filter=package_name=={args.operator_name}",
-        ),
-        args.cert_path,
```

Comment:
So, should we delete those args from argparser now?

Reply:
Removed